### ISSUE TITLE
Download form links should say that they will open in a new window & include icon

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95529]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

It should be similar to the experience when you land on "Please read and accept the privacy policy" link
![Image](https://github.com/user-attachments/assets/65d15fb7-c081-48a4-afb7-c163c5e44a6c)

![Image](https://github.com/user-attachments/assets/202cf630-3a77-40d0-ae41-f850512255df)

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-3
**Page**: confirmation
**Environment**: any screenreader/browser combo
**Estimated Level of Effort to fix**: low
**Design system component involved**: No
**Relevant WCAG success criteria**: [3.2.5 Change on Request Level AAA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview#change-on-request)

### To Reproduce:
- Use a screenreader and navigate to the Download form links

#### Expected Behavior
-  You should see the new window icon + hear it announced by screenreader

#### Actual Behavior
- User does not realize that clicking on the Download link will open in a new window, and they lose their context. 

## Tasks
- [ ] Update the link
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Screenreader users hear that the links will open in a new window
- [ ] Accessibility Audit issue tracking doc is updated with ticket status

**Comment by bacitracin** [comment]
This issue has been submitted as an experimental design in this ticket so closing this ticket https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3537
